MergerLogic/Utils/OneXOneConvetor.cs

```diff
@@ -0,0 +1,139 @@
+﻿using MergerLogic.Batching;
+using MergerLogic.DataTypes;
+
+namespace MergerLogic.Utils
+{
+    public class OneXOneConvetor
```

Comment:
Add comment explaining the issue with these functions, and a TODO to add tests for this case.